clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/HostLevelIngestionStats.java

```diff
@@ -163,19 +164,14 @@ public HostLevelIngestionStats(
       HostLevelIngestionStats totalStats,
       Map<String, StoreIngestionTask> ingestionTaskMap) {
     super(metricsRepository, storeName);
+
+    boolean isSystemStore = VeniceSystemStoreUtils.isSystemStore(storeName);
```

Comment:
I think it's ok to remove backup and future version stats for all system stores, but I think we still need current version ingestion metrics.